docs/touchableopacity.md

```diff
@@ -133,23 +133,23 @@ Inherits [TouchableWithoutFeedback Props](touchablewithoutfeedback.md#props).
 
 ### `style`
 
-| Type       | Required |
-| ---------- | -------- |
-| View.style | No       |
+| Type       |
+| ---------- |
+| View.style |
```

Comment:
Can you link this type to the page below:
* https://reactnative.dev/docs/next/view-style-props